docs/tutorials/porch-development-environment/README.md

```diff
@@ -0,0 +1,149 @@
+
+- The `porch-system` namespace, an API Service called `apiservice.apiregistration.k8s.io/v1alpha1.porch.kpt.dev` and the `service.api` service to expose the API Service. These resources are defined in the the file `deployments/local/localconfig.yaml`
+- the `repositories.config.porch.kpt.dev` and `functions.config.porch.kpt.dev` CRDs. These CRDs are defined in the `api/porchconfig/v1alpha1/` directory.
+- the `packagerevs.config.porch.kpt.dev` CRD. This CRD is defined in the `internal/api/porchinternal/v1alpha1/` directory.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/nephio-project/porch/main/deployments/local/localconfig.yaml
+kubectl apply -f https://raw.githubusercontent.com/nephio-project/porch/main/api/porchconfig/v1alpha1/config.porch.kpt.dev_repositories.yaml
+kubectl apply -f https://raw.githubusercontent.com/nephio-project/porch/main/api/porchconfig/v1alpha1/config.porch.kpt.dev_functions.yaml
+kubectl apply -f https://raw.githubusercontent.com/nephio-project/porch/main/internal/api/porchinternal/v1alpha1/config.porch.kpt.dev_packagerevs.yaml
+```
+Verify that the resources have been created
+```
+kubectl api-resources | grep -i porch
+functions                                      config.porch.kpt.dev/v1alpha1     true         Function
+packagerevs                                    config.porch.kpt.dev/v1alpha1     true         PackageRev
+packagevariants                                config.porch.kpt.dev/v1alpha1     true         PackageVariant
+packagevariantsets                             config.porch.kpt.dev/v1alpha2     true         PackageVariantSet
+repositories                                   config.porch.kpt.dev/v1alpha1     true         Repository
+```
+
+# Configure VSCode to run the Porch server
+
+Check out porch and start vscode in the root of your checked out Porch repo.
+
+Edit your local `.vscode.launch.json` file as follows:
+1. Change the `--kubeconfig` value to point at your local Kind cluster configuration file
+2. Change the `--function-runner` IP address to that of the function runner service running in the Kind `management` cluster
+
+```
+        {
+            "name": "Launch Server",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceFolder}/cmd/porch/main.go",
+            "args": [
+                "--secure-port=9443",
+                "--v=7",
+                "--standalone-debug-mode",
+                "--kubeconfig=/home/sean-citizen/.kube/kind-management-config",
+                "--cache-directory=${workspaceFolder}/.cache",
+                "--function-runner=172.18.255.201:9445"
+            ],
+            "env": {
+				"KUBECONFIG": "/Users/liam/.kube/kind-management-config"
+			},
+            "cwd": "${workspaceFolder}"
+        },
+```
+
+You can now launch the Porch server locally in VSCode by selecting the "Launch Server" configuration on the VSCode "Run and Debug" window.
```

Comment:
Might be no harm to link to a guide for this.
https://code.visualstudio.com/docs/editor/debugging